src/briefcase/console.py

```diff
@@ -960,3 +960,45 @@ def selection_question(
         )
 
         return ordered[int(index) - 1][0]
+
+    def boolean_question(
+        self,
+        description,
+        intro: str,
+        default: bool | None = None,
+        override_value: bool | None = None,
+    ) -> bool:
+        """Ask the user a boolean question who's answer requires selecting yes/no
+
+        :param description: A short description of the question being asked. This text
+            is used in prompts and a header bar prefacing the question.
+        :param intro: An introductory paragraph explaining the question being asked.
+        :param default: The default option for empty user input.
+        :param override_value: A pre-selected answer for the question. This can be used
+            to shortcut asking the question, such as when a command line option provides
+            a value. If provided and valid, the header bar will be displayed, but the
+            intro paragraph and option list will not.
+        :returns: The user's chosen answer or none if closed without input
+        """
+        if default not in {True, False, None}:
+            raise ValueError(
+                f"'{default}' is not a valid default value for {description}? Must be True, False, or None."
```

Comment:
Adding this handling is a good instinct, but not required in this case. The type declaration forces this to True/False/None; plus, at runtime, Python will duck type the value. Anything that quacks like a True value will behave like "True".